projects/@shared/lib/validate-data.js

```diff
@@ -1,30 +1,64 @@
+import getImageDimensions from './get-image-dimensions';
+
 // Docs: https://joi.dev/api/?v=17.4.0#anyvalidatevalue-options
 const JOI_VALIDATION_OPTIONS = {
   abortEarly: false,
   convert: false,
 };
 
-const validateData = (data, schema) => {
-  if (!data.length) {
-    return { errors: 'No data to validate' };
-  }
+/**
+ * Transforms the data array into an object that Joi can validate.
+ *
+ * @param {Array} data
```

Comment:
Could you add the `@returns` annotation as well?

Reply:
Done.